rpm_spec/manageiq.spec.in

```diff
@@ -47,18 +47,12 @@ pathfix.py -pni "%{__python3} %{py3_shbang_opts}" %{gemset_builddir}
 %build
 cd %{_builddir}
 
-cat <<"EOF" > %{gemset_builddir}/enable
-export APPLIANCE="true"
-export BUNDLE_GEMFILE=%{app_root}/Gemfile
-export GEM_HOME=%{gemset_root}
-export GEM_PATH=%{gemset_root}:$(gem env path)
-export PATH=%{gemset_root}/bin:$PATH
-EOF
-
-cat <<"EOF" > %{appliance_builddir}/LINK/etc/default/evm_production
-export APPLIANCE_SOURCE_DIRECTORY=%{appliance_root}
-export APPLIANCE_TEMPLATE_DIRECTORY=${APPLIANCE_SOURCE_DIRECTORY}/TEMPLATE
-source %{gemset_root}/enable
+cat <<"EOF" > %{appliance_builddir}/LINK/etc/default/production.properties
+APPLIANCE_SOURCE_DIRECTORY=%{appliance_root}
+APPLIANCE_TEMPLATE_DIRECTORY=%{appliance_root}/TEMPLATE
+GEM_HOME=%{gemset_root}
+GEM_PATH=%{gemset_root}:/usr/share/gems:/usr/local/share/gems
```

Comment:
@kbrock Did you verify this is the same as `gem env path` in both appliances and podified?  If so, I am good with this.

Reply:
I am taking the approach that we will not use any gems except very low level system gems. These are installed using rpm and seems slim that they would change location

```
rubygem-json-2.1.0-106.module_el8.1.0+249+93480f15.x86_64
rubygem-openssl-2.1.2-106.module_el8.1.0+249+93480f15.x86_64
rubygem-bigdecimal-1.4.1-106.module_el8.1.0+249+93480f15.x86_64
rubygem-did_you_mean-1.3.0-106.module_el8.1.0+249+93480f15.noarch
rubygem-irb-1.0.0-106.module_el8.1.0+249+93480f15.noarch
rubygem-psych-3.1.0-106.module_el8.1.0+249+93480f15.x86_64
rubygem-rake-12.3.2-106.module_el8.1.0+249+93480f15.noarch # we use a later version
rubygem-io-console-0.4.7-106.module_el8.1.0+249+93480f15.x86_64
rubygem-rdoc-6.1.0-106.module_el8.1.0+249+93480f15.noarch
```

I did not verify pods but I will do that